i2s tests fail to build for sam_v71_xult board

**Describe the bug**
I2S tests fail to build for the `sam_v71_xult` board. There are a few issues present here:
- Overlays for the `sam_v71_xult/samv71q21b` target are missing
- I2S speed test does not seem to be supported, but is run by twister

Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using?
   Builds are failing on `sam_v71_xult/samv71q21` and `sam_v71_xult/samv71q21b` targets
 - What have you tried to diagnose or workaround this issue?
   Creating overlays for the `sam_v71_xult/samv71q21b`, this still does not fix the I2S speed test
 - Is this a regression? If yes, have you been able to "git bisect" it to a
   specific commit?
   Seems to be introduced by transition to HWMv2 for the `sam_v71_xult/samv71q21b`. Less clear on the I2S speed test issue

**To Reproduce**
Steps to reproduce the behavior:
1 ./scripts/twister -p sam_v71_xult/samv71q21 -T tests/drivers/i2s/
2. ./scripts/twister -p sam_v71_xult/samv71q21b -T tests/drivers/i2s/

**Impact**
Blocking CI on #69833

**Logs and console output**
Same error for I2S speed and I2S API test. Seems like a case of the `xdmac` node not being enabled
```
/home/danieldegrasse/zephyr/zephyrproject/zephyr/include/zephyr/device.h:89:41: error: '__device_dts_ord_48' undeclared here (not in a function); did you mean '__device_dts_ord_18'?
   89 | #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id)
      |                                         ^~~~~~~~~
/home/danieldegrasse/zephyr/zephyrproject/zephyr/include/zephyr/toolchain/common.h:137:26: note: in definition of macro '_DO_CONCAT'
  137 | #define _DO_CONCAT(x, y) x ## y
      |                          ^
/home/danieldegrasse/zephyr/zephyrproject/zephyr/include/zephyr/device.h:89:33: note: in expansion of macro '_CONCAT'
   89 | #define DEVICE_NAME_GET(dev_id) _CONCAT(__device_, dev_id)
      |                                 ^~~~~~~
/home/danieldegrasse/zephyr/zephyrproject/zephyr/include/zephyr/device.h:216:37: note: in expansion of macro 'DEVICE_NAME_GET'
  216 | #define DEVICE_DT_NAME_GET(node_id) DEVICE_NAME_GET(Z_DEVICE_DT_DEV_ID(node_id))
      |                                     ^~~~~~~~~~~~~~~
/home/danieldegrasse/zephyr/zephyrproject/zephyr/include/zephyr/device.h:233:34: note: in expansion of macro 'DEVICE_DT_NAME_GET'
  233 | #define DEVICE_DT_GET(node_id) (&DEVICE_DT_NAME_GET(node_id))
      |                                  ^~~~~~~~~~~~~~~~~~
/home/danieldegrasse/zephyr/zephyrproject/zephyr/drivers/i2s/i2s_sam_ssc.c:1017:20: note: in expansion of macro 'DEVICE_DT_GET'
 1017 |         .dev_dma = DEVICE_DT_GET(DT_INST_DMAS_CTLR_BY_NAME(0, tx)),
```

--- a/tests/drivers/i2s/i2s_api/boards/sam_e70_xplained_same70q21b.overlay
+++ b/tests/drivers/i2s/i2s_api/boards/sam_e70_xplained_same70q21b.overlay
@@ -1,0 +1,11 @@
+/* i2s-node0 is the transmitter/receiver */
+
+/ {
+	aliases {
+		i2s-node0 = &ssc;
+	};
+};
+
+&xdmac {
+	status = "okay";
+};

--- a/tests/drivers/i2s/i2s_api/boards/sam_v71_xult_samv71q21b.overlay
+++ b/tests/drivers/i2s/i2s_api/boards/sam_v71_xult_samv71q21b.overlay
@@ -1,0 +1,11 @@
+/* i2s-node0 is the transmitter/receiver */
+
+/ {
+	aliases {
+		i2s-node0 = &ssc;
+	};
+};
+
+&xdmac {
+	status = "okay";
+};

--- a/tests/drivers/i2s/i2s_speed/boards/sam_e70_xplained_same70q21.overlay
+++ b/tests/drivers/i2s/i2s_speed/boards/sam_e70_xplained_same70q21.overlay
@@ -1,0 +1,11 @@
+/* i2s-node0 is the transmitter/receiver */
+
+/ {
+	aliases {
+		i2s-node0 = &ssc;
+	};
+};
+
+&xdmac {
+	status = "okay";
+};

--- a/tests/drivers/i2s/i2s_speed/boards/sam_e70_xplained_same70q21b.overlay
+++ b/tests/drivers/i2s/i2s_speed/boards/sam_e70_xplained_same70q21b.overlay
@@ -1,0 +1,11 @@
+/* i2s-node0 is the transmitter/receiver */
+
+/ {
+	aliases {
+		i2s-node0 = &ssc;
+	};
+};
+
+&xdmac {
+	status = "okay";
+};

--- a/tests/drivers/i2s/i2s_speed/boards/sam_v71_xult_samv71q21.overlay
+++ b/tests/drivers/i2s/i2s_speed/boards/sam_v71_xult_samv71q21.overlay
@@ -1,0 +1,11 @@
+/* i2s-node0 is the transmitter/receiver */
+
+/ {
+	aliases {
+		i2s-node0 = &ssc;
+	};
+};
+
+&xdmac {
+	status = "okay";
+};

--- a/tests/drivers/i2s/i2s_speed/boards/sam_v71_xult_samv71q21b.overlay
+++ b/tests/drivers/i2s/i2s_speed/boards/sam_v71_xult_samv71q21b.overlay
@@ -1,0 +1,11 @@
+/* i2s-node0 is the transmitter/receiver */
+
+/ {
+	aliases {
+		i2s-node0 = &ssc;
+	};
+};
+
+&xdmac {
+	status = "okay";
+};

